src/NCronJob/Registry/JobRegistry.cs

```diff
@@ -86,12 +140,26 @@ private void AssertNoDuplicateJobNames(string? additionalJobName = null)
         }
     }
 
+    public void UpdateJobDefinitionsToRunAtStartup<TJob>()
+    {
+        foreach (var jobDefinition in JobDefinitions)
+        {
+            if (jobDefinition.Type != typeof(TJob))
+            {
+                continue;
+            }
+
+            jobDefinition.IsStartupJob = true;
+        }
+    }
+
     private sealed class JobDefinitionEqualityComparer : IEqualityComparer<JobDefinition>
```

Comment:
In principle that is absolutely okay. It would be nice just to have `Equals` do the right thing inside `JobDefinition` - given that it is already a `record`.

Doesn't that work? I feel we used that somewhere else. But could br wrong

Reply:
Yes! ;D